docs/developing_iree/e2e_benchmarking.md

```diff
@@ -275,9 +275,24 @@ Android.
 The first two are to build it directly, either in a
 [`docker` container](https://www.tensorflow.org/lite/guide/build_android#set_up_build_environment_using_docker)
 or
-[in your own environment](https://www.tensorflow.org/lite/guide/build_android#set_up_build_environment_without_docker). Assuming you can build
-TensorFlow with Android, you can configure the TFLite `benchmark_model` binary
-in the following ways:
+[in your own
+environment](https://www.tensorflow.org/lite/guide/build_android#set_up_build_environment_without_docker).
+To build TensorFlow tools with Android:
+
+- Run `./configure` under TensorFlow repo.
+- Add below section to TensorFlow WORKSPACE file.
```

Comment:
Add the following section to the TensorFlow WORKSPACE file.

Reply:
done